sourmash/lca/lca_utils.py

```diff
@@ -382,7 +382,11 @@ def _signatures(self):
 
         for (k, v) in self.hashval_to_idx.items():
             for vv in v:
-                temp_vals[vv].append(k)
+                temp_hashes = temp_vals[vv]
+                temp_hashes.append(k)
+                if len(temp_hashes) > 50:
+                    sigd[vv].add_many(temp_hashes)
+                    temp_vals[vv].clear()
```

Comment:
sigh, I hate you Python 2... `.clear()` is not available in Python 2.
```suggestion
                    del temp_vals[vv]
```